detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnnecessaryParenthesesSpec.kt

```diff
@@ -571,6 +579,44 @@ class UnnecessaryParenthesesSpec {
         assertThat(testCase.rule.compileAndLint(code)).hasSize(if (testCase.allowForUnclearPrecedence) 4 else 5)
     }
 
+    @ParameterizedTest
+    @MethodSource("cases")
+    fun `float closed range without integer part`(testCase: RuleTestCase) {
+        val code = """
+            val a = .1F..(.2F)
+        """.trimIndent()
+        assertThat(testCase.rule.compileAndLint(code)).hasSize(if (testCase.allowForUnclearPrecedence) 0 else 1)
```

Comment:
```suggestion
        assertThat(testCase.rule.compileAndLint(code)).isEmpty()
```

These parenthesis are always needed, right? It's not a fact of clear or unclear.